bundles/org.openhab.binding.nibeheatpump/src/main/java/org/openhab/binding/nibeheatpump/internal/handler/NibeHeatPumpHandler.java

```diff
@@ -411,7 +411,7 @@ private void parseWriteEnabledItems() throws IllegalArgumentException {
         logger.debug("Enabled registers for write commands: {}", itemsToEnableWrite);
     }
 
-    private State convertNibeValueToState(NibeDataType dataType, double value, String acceptedItemType) {
+    private State convertNibeValueToState(VariableInformation variableInfo, int value, String acceptedItemType) {
```

Comment:
Maybe we should add junit tests for this. WDYT?

Reply:
I actually did a test to verify this code, but method is private and have no idea how to test it without making it at least protected.

Reply:
Reflection helped =) Learning every day.